app/account/migrations/0001_initial.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.0.4 on 2018-04-20 16:15
+
+import app.storage
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django_extensions.db.fields
+import economy.models
+import storages.backends.s3boto3
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('dashboard', '0056_profile_avatar'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Organization',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_on', models.DateTimeField(db_index=True, default=economy.models.get_time)),
+                ('modified_on', models.DateTimeField(default=economy.models.get_time)),
+                ('avatar', models.ImageField(storage=storages.backends.s3boto3.S3Boto3Storage(), upload_to=app.storage.get_avatar_path)),
+                ('description', models.TextField(blank=True)),
+                ('gh_repos_data', django.contrib.postgres.fields.jsonb.JSONField(default={})),
+                ('gh_user_data', django.contrib.postgres.fields.jsonb.JSONField(default={})),
+                ('name', models.CharField(max_length=255)),
+                ('slug', django_extensions.db.fields.AutoSlugField(blank=True, editable=False, populate_from='name')),
+                ('followers', models.ManyToManyField(related_name='follows_org', to='dashboard.Profile')),
+                ('members', models.ManyToManyField(related_name='organizations', to='dashboard.Profile')),
```

Comment:
`members` ? Would this be of any value ?

Reply:
In this case, any `dashboard.models.Profile` can be added as a member of any given `Organization`